fluent/client/ws/connection.go

```diff
@@ -70,17 +94,19 @@ func NewConnection(conn ext.Conn, opts ConnectionOptions) (Connection, error) {
 	}
 
 	if opts.ReadHandler == nil {
-		opts.ReadHandler = func(c Connection, _ int, _ []byte, err error) error {
-			if err != nil {
-				_ = c.Close()
-			}
-
+		opts.ReadHandler = func(_ Connection, _ int, _ []byte, err error) error {
```

Comment:
Is there no need for the default handler to close the connection in case of an error is given?

Reply:
Talked to Eric